pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -55,9 +55,14 @@ type PipelineRunSpec struct {
 	// Params is a list of parameter names and values.
 	Params []Param `json:"params,omitempty"`
 	// +optional
-	ServiceAccount string `json:"serviceAccount"`
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
+	// DeprecatedServiceAccount is a depreciated alias for ServiceAccountName.
+	// Deprecated: Use serviceAccountName instead.
 	// +optional
-	ServiceAccounts []PipelineRunSpecServiceAccount `json:"serviceAccounts,omitempty"`
+	DeprecatedServiceAccount string `json:"serviceAccount,omitempty"`
+	// +optional
+	DeprecatedServiceAccounts []DeprecatedPipelineRunSpecServiceAccount `json:"serviceAccounts,omitempty"`
+	ServiceAccountNames       []PipelineRunSpecServiceAccountName       `json:"serviceAccountNames,omitempty"`
```

Comment:
Should `ServiceAccountNames` also be marked `// +optional`?

Reply:
yup.